assets/files/etc/haproxy/members_healthcheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec wget -q --no-check-certificate "https://$3:6443/healthz"
```

Comment:
```suggestion
exec wget -q --no-check-certificate "https://$3:6443/readyz"
```

Reply:
Done

Reply:
Finally this endpoint does not seem to be there